operations/service.go

```diff
@@ -106,8 +107,33 @@ func Service() cli.Command {
 				return errors.Wrap(err, "problem getting application configuration")
 			}
 
-			var d depot.Depot
+			var um gimlet.UserManager
```

Comment:
I kind of liked keeping more stuff out of operations/service (and I made a pass a while ago to factor it down.) It feels tempting to put a lot of the setup code here, but the more code we push into the packages the better. Because we end up making the user manager field in the service public, as a result of this change, we can just create it in the rest service and then reference it later. 

Another option would be create a function that generates the user manager in the service package (which would make more sense to test.) We could also just have two of them which would probably up our traffic to LDAP, but might be more logical seeming.

Reply:
Sounds good. It is safe to assume that if gRPC is supporting TLS auth, there is either LDAP or naive auth configured, correct?

Reply:
yeah.